framework/UPGRADE.md

```diff
@@ -17,6 +17,17 @@ php composer.phar global require "fxp/composer-asset-plugin:~1.1.0"
 Upgrade from Yii 2.0.6
 ----------------------
 
+* Added new requirement: ICU Data version >= 49.1. Please, ensure that your environment has ICU data installed and
+up to date to prevent unexpected behavior or crashes.
+
+ > Tip: The easiet way to check: `php --info | grep ICU`
```

Comment:
It's not:

1. We have requirements script.
2. The common mistake is to use different `php.ini` in CLI / fpm / apache module.